cfg/rationalize.go

```diff
@@ -25,8 +37,39 @@ func decodeURL(u string) (string, error) {
 	return decodedURL.String(), nil
 }
 
+// resolveMetadataCacheTTL returns the ttl to be used for stat/type cache based
+// on the user flags/configs.
+func resolveMetadataCacheTTL(v isSet, c *MetadataCacheConfig) {
+	// If metadata-cache:ttl-secs has been set, then it overrides both
+	// stat-cache-ttl and type-cache-tll.
+	if v.IsSet("metadata-cache.ttl-secs") {
```

Comment:
The key "metadata-cache.ttl-secs" and "metadata-cache.stat-cache-max-size-mb" are being repeated at multiple places. Shall we create constants for them?